contributing/README.md

```diff
@@ -0,0 +1,150 @@
+# Contributing to WireMock
+
+[![a](https://img.shields.io/badge/slack-Join%20us-brightgreen?style=flat&logo=slack)](http://slack.wiremock.org/)
+
+WireMock exists and continues to thrive due to the efforts of over 150 contributors,
+and we continue to welcome contributions to its evolution.
+Regardless of your expertise and time you could dedicate,
+there're opportunities to participate and help the project!
+
+## Ways to contribute
+
+You can help make WireMock a better tool in a number of ways:
+
+- Help other WireMock users and sharing tips
+- Write or improve documentation
+- Raise an issue if you discover a bug
+- Contribute bug fixes, new features or enhancements
+- Spread the word about WireMock, share your stories and contributions
+
+## Helping users
+
+- Join the [community Slack channel](http://slack.wiremock.org/),
+  especially the `#help` channel we use for random Q&A
+- Subscribe to the [`#wiremock` tag on StackOverflow](https://stackoverflow.com/questions/tagged/wiremock).
+  Many questions are raised there, and it is a good opportunity to get StackOverflow medals :smile:
+- If you have expertise with a particular WireMock component, e.g. _WireMock Java_ or _WireMock.NET_,
```

Comment:
Let's leave .NET out of the picture until we've sorted out our relationship with that project.

Reply:
```suggestion
- If you have expertise with a particular WireMock component, e.g. _WireMock Java_ or the _Python WireMock Client_,
```

Agreed